client/hotrod-client/src/main/java/org/infinispan/client/hotrod/configuration/ConfigurationBuilder.java

```diff
@@ -282,6 +285,12 @@ public ConfigurationBuilder maxRetries(int maxRetries) {
    }
 
    @Override
+   public ConfigurationBuilder javaSerialWhiteLists(String... regExs) {
```

Comment:
Also this method should be named something like `add...` since we aren't overriding existing white list with subsequent calls.